### ISSUE TITLE
mid-word inline italics

### ISSUE THREAD

**Issue by ckoppelman** [Rosey/markdown-draft-js#53]
If I type into the draft editor: abcXYZabc where XYZ is italic , it will render the markdown as `abc_XYZ_abc` (this is fine).  When that gets converted back into DraftJS, the text is rendered as `abc_XYZ_abc`.

**Comment by Rosey** [comment]
Yeah this is sort of a... hm. I've noticed this before and I'm not sure what the best way to fix it is. It seems markdown doesn't work like that, eg if I type this in here on github: abc_XYZ_abc it won't be italicized. There has to be a _space_ **before** the \_ symbol for it to get picked up. But by inserting a space when converting from draft to markdown, we are changing the content which may also not be desired.

So it seems the options are:

1. Insert space, which might be bad
2. Don't italicize in this case, which sadly will also confuse people 😢

**Comment by Rosey** [comment]
Tentatively labeling as a wontfix pending further discussion on the best course of action 😄

**Comment by ckoppelman** [comment]
Perhaps using `*` for emphasis is the best approach.  Intraword underscores are not respected, but intraword asterisks are.  (I just learned this this moment).
